### DISCUSSION TITLE
reboot/reset

### DISCUSSION THREAD

**Discussion by jschoch** [multigcs/LinuxCNC-RIO#43] [Q&A]
is there a way to map a button to reset/reboot the FPGA?

side question:  wonder if there is a way to save the state and load it on reboot?

**Comment by multigcs** [comment]
reset for what ? it's not a CPU :)
the artyA7 board have a reload/reconfigure button

save state : not in the FPGA, not planned

**Comment by jschoch** [comment]
maybe exposing the reset pin on the W5500 and then tying that to estop is a better idea?
